treenode/models.py

```diff
@@ -204,11 +204,18 @@ def get_display(self, indent=True, mark='— '):
     def get_display_text(self):
         """
         Gets the text that will be indented in `get_display` method.
-        Returns the `treenode_display_field` value by default.
+        Returns the `treenode_display_field` value if specified,
+        otherwise falls back on the model's __str__().
         Override this method to return another field or a computed value. #27
         """
-        field_name = getattr(self, 'treenode_display_field', 'pk')
-        text = getattr(self, field_name)
+        if hasattr(self, 'treenode_display_field') and self.treenode_display_field is not None:
+            field_name = getattr(self, 'treenode_display_field', 'pk')
+            text = getattr(self, field_name)
+        elif type(self).__str__ is not object.__str__:
+            text = f'{self}'
+        else:
+            raise ValueError(
+                f'Neither __str__ nor treenode_display_field are defined for model')
```

Comment:
There is no need to use an f-string here.

Reply:
ah yeah lol